Gruntfile.js

```diff
@@ -81,10 +81,19 @@ module.exports = function (grunt) {
         autoWatch: false
       },
       browsers: {
-        browsers: ['Chrome', 'Firefox']
+        browsers: [ 'Chrome', 'Firefox' ]
       },
       phantom: {
-        browsers: ['PhantomJS']
+        browsers: [ 'PhantomJS' ]
+      },
+      integration: {
+        browsers: [ 'Chrome', 'Firefox' ],
+        options: { files: [
```

Comment:
having this in defined both in karma.conf.js for most tests, but here for this one seems non-ideal.